CHANGELOG.md

```diff
@@ -18,6 +18,9 @@
 - keep track of securejoin joiner status in database to survive restarts #2920
 - remove never used `SentboxMove` option #3111
 
+### Improvements
+- Improve speed
```

Comment:
```suggestion
- improve speed by caching config values
```

i'd also drop the headline, we usually only have _API Changes_, _(Other) Changes_ and _Fixes_